src/common/types/value.cpp

```diff
@@ -381,18 +389,22 @@ std::string Value::toString() const {
     }
     default:
         throw NotImplementedException("Value::toString for type " +
```

Comment:
I think our convension for NotImplmenetedException is ClassName::MethodName? So change to `Value::toString()`?